components/admin_console/admin_definition.jsx

```diff
@@ -1507,17 +1512,42 @@ export default {
                                     if (job.data.update_count && job.data.update_count.length > 0) {
                                         updateCount = job.data.update_count;
                                     }
+
+                                    if (job.data.mattermost_groups_count) {
+                                        mattermostGroups = job.data.mattermost_groups_count;
+                                    }
+
+                                    if (job.data.ldap_groups_count) {
+                                        ldapGroups = job.data.ldap_groups_count;
+                                    }
+
+                                    if (job.data.group_delete_count) {
+                                        groupDeleteCount = job.data.group_delete_count;
+                                    }
+
+                                    if (job.data.group_member_delete_count) {
+                                        groupMemberDeleteCount = job.data.group_member_delete_count;
+                                    }
+
+                                    if (job.data.group_member_add_count) {
+                                        groupMemberAddCount = job.data.group_member_add_count;
+                                    }
                                 }
 
                                 return (
                                     <FormattedMessage
```

Comment:
Should this be `FormattedMarkdownMessage`?

Reply:
I was thinking that too but ended up going with CSS but now that you mention it too let me switch to `FormattedMarkdown` and see if it's cleaner because I'm thinking you're right.